.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You can add this file to .gitignore as it is not correct to add Ide settings over here.